airflow/www/static/js/dag/details/taskInstance/Logs/utils.test.tsx

```diff
@@ -40,6 +40,7 @@ const mockTaskLog = `
 [2022-06-04 00:00:02,010] {taskinstance.py:1548} WARNING - Exporting env vars: AIRFLOW_CTX_DAG_OWNER=*** AIRFLOW_CTX_DAG_ID=test_ui_grid
 [2024-07-01 00:00:02,010] {taskinstance.py:1548} INFO - Url parsing test => "https://apple.com", "https://google.com", https://something.logs/_dashboard/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-1d,to:now))&_a=(columns:!(_source),filters:!(('$state':(store:appState))))
 `;
+const mockExtraLog = `--------------------------------------------------------------------------------`;
```

Comment:
Can we add an example with a stack trace for example that has intermediate lines, (typical case where only the first line is considered ERROR, while the entire trace should be).

I don't understand what this `mockExtraLog` is supposed to do.